client/app/admin/admin.controller.js

```diff
@@ -13,7 +13,38 @@ angular.module('observatory3App')
             // Use the User $resource to fetch all users
 
             $scope.users = [];
-            $scope.users = User.allstats();
+            User.allstats({},function(users){
+                $scope.users = users;
+
+                for (var a = 0; a < $scope.users.length; a++){
```

Comment:
this processing could be simplified into a function like countUserAttendance() would be a lot cleaner imo